docs/components/console/manage-clusters/cluster-backups.md

```diff
@@ -0,0 +1,45 @@
+---
+id: create-backups
+title: Create backup
+description: "If your organization works within Camunda's Enterprise plan, you can create cluster backups."
+---
+
+If your organization works within Camunda's [Enterprise](https://camunda.com/enterprise/) plan, you can create a manual and scheduled [backups](/components/concepts/backups.md) of your cluster.
+
+Only the three most recent successful backups of each type are kept, meaning you can have three manual and three scheduled backups.
+
+## Create a manual backup
+
+You can create a manual backup every five hours.
+
+To create a manual backup, take the following steps:
+
+1. Select the **Backups** tab.
+2. Click **Create manual backup**.
+
+![cluster-details](./img/cluster-detail-backups.png)
+
+3. A popup model will appear with more information about manual backups, including retention. Click **Create backup**.
+
+![cluster-details](./img/cluster-detail-backups-manual.png)
+
+:::note
+Only the three most recent successful manual backups are kept. If you already have three manual backups, the oldest manual backup is automatically removed.
```

Comment:
On line 9 we state

> Only the three most recent successful backups of each type are kept, meaning you can have three manual and three scheduled backups.

But this feels like duplication. Are we just trying to stress the importance of this? If so, I might just consolidate the two in an admonition, like

> Only the three most recent successful manual backups of each type are kept, meaning you can have three manual and three scheduled backups. If you already have three manual backups, the oldest manual backup is automatically removed.

Reply:
Exactly—we will need to monitor this closely. I like your change and will add it!

Reply:
I added one consolidated caution admonition at the top of the file.